docs/api/geo_datasets.csv

```diff
@@ -1,17 +1,18 @@
 Dataset,Type,Source,Size (px),Resolution (m)
-`Aboveground Woody Biomass`_,Mask,"Landsat, LiDAR","~40,000x40,000",~30
+`Aboveground Woody Biomass`_,Mask,"Landsat, LiDAR","40,000x40,000",30
 `Aster Global DEM`_,Mask,Aster,"3,601x3,601",30
-`Canadian Building Footprints`_,Labels,Generated,,
-`Chesapeake Land Cover`_,"Imagery, Labels",,,1
```

Comment:
I think "mask" is better than "labels" for semantic segmentation problems

Reply:
Would go with "Masks"